content/en/blog/releases/kubecf-1.0.1.md

```diff
@@ -0,0 +1,58 @@
+
+---
+title: "KubeCF Release v1.0.1"
+linkTitle: "KubeCF Release v1.0.1"
+date: 2020-03-19
+description: >
+  KubeCF 1.0.1 has been released!
+---
+
+# Bug fixes
+
+- Fixes the permissions issue in autoscaler db when the pod restarts. [#467](https://github.com/cloudfoundry-incubator/kubecf/pull/467)
+- Moved pxc image to cfcontainerization organization, like all the other images. [#479](https://github.com/cloudfoundry-incubator/kubecf/pull/479)
+- The database-seeder job pod was not being deleted after KubeCF helm installation was removed with success. [#493](https://github.com/cloudfoundry-incubator/kubecf/pull/493)
+- Security group setups. [#485](https://github.com/cloudfoundry-incubator/kubecf/pull/485)
```

Comment:
Are these missing? https://github.com/cloudfoundry-incubator/kubecf/pull/485 @fargozhu

Reply:
@jimmykarily missing from where? The #485 was merged and shipped on 1.0.1

Reply:
@fargozhu #485 fixes: #475 and #480 (according to the PRs description). If those issue are fixed, should they be listed here, in the bug fixes of the 1.0.1 release?